### PR TITLE
Add size patch for mesh size function

### PR DESCRIPTION
**Pull request by SorooshMani-NOAA**

### CODE REVIEW

File: geomesh/hfun/raster.py [hunk 1]
```diff
@@ -260,6 +261,117 @@ def msh_t(self, window: rasterio.windows.Window = None,
...
+        if target_size is None:
+            raise ValueError('Argument target_size must be specified if no '
+                             'global hmin has been set.')
+        if target_size <= 0:
+            raise ValueError("Argument target_size must be greater than zero.")
+
+        # For expansion_rate
+        if expansion_rate != None:
+            exteriors = [ply.exterior for ply in multipolygon]
+            for exterior in exteriors:
```
Comment (jreniel) [inline geomesh/hfun/raster.py:297]: Should we do the same for the interiors? The fastest way to use `add_feature()` is to collect all the features on a single `MultiLineString` object.

**Comment by JaimeCalzadaNOAA** [comment]
A work of art, Thanks!
Let me know once you feel confident that this should go into the main branch.

File: geomesh/hfun/raster.py (hunk 1 above)
Comment (SorooshMani-NOAA) [reply geomesh/hfun/raster.py:297]: OK, I'll add a fix and push

**Comment by SorooshMani-NOAA** [comment]
Thanks, I'll add the fix for the code review, do one more round of testing and ping you once I think it's good to go in.

**Pull request merged**